### PR TITLE
Feature messenger

### PR DESCRIPTION
**Pull request by marceldobrica**

### CODE REVIEW

File: .env [hunk 1]
```diff
@@ -33,7 +33,7 @@ MESSENGER_TRANSPORT_DSN=doctrine://default?auto_setup=0
 
 ###> symfony/messenger ###
 # Choose one of the transports below
```
Comment (ovidiuenache) [inline .env]: Remove the useless comments

File: config/packages/messenger.yaml [hunk 2]
```diff
@@ -1,24 +1,15 @@
...
+                    queue_name: sms
 
         routing:
-            Symfony\Component\Mailer\Messenger\SendEmailMessage: sync
-            Symfony\Component\Notifier\Message\ChatMessage: sync
-            Symfony\Component\Notifier\Message\SmsMessage: sync
-
-            # Route your messages to the transports
-            # 'App\Message\YourMessage': async
+            Symfony\Component\Mailer\Messenger\SendEmailMessage: async_mail
```
Comment (ovidiuenache) [inline config/packages/messenger.yaml:14]: I assume you have the infinite loop issue. Did you test this? Are emails sent?

File: src/HttpClient/HttpClientSmsUser.php [hunk 3]
```diff
@@ -0,0 +1,49 @@
...
+            'body' => $message
+        ];
+        $postJson = json_encode($postData);
+
+        $this->client->request(
+            Request::METHOD_POST,
+            '',
+            [
+                'headers' => [
+                    'Content-Type: application/json',
```
Comment (ovidiuenache) [inline src/HttpClient/HttpClientSmsUser.php]: These are always the same. You can set them in the scoped clients configuration.

File: src/HttpClient/HttpClientSmsUser.php [hunk 4]
```diff
@@ -0,0 +1,49 @@
...
+    {
+        $postData = [
+            'receiver' => $phone,
+            'body' => $message
+        ];
+        $postJson = json_encode($postData);
+
+        $this->client->request(
+            Request::METHOD_POST,
+            '',
```
Comment (ovidiuenache) [inline src/HttpClient/HttpClientSmsUser.php]: why empty string? It should be /api/messages or /messages

File: src/HttpClient/HttpClientImportPogramme.php [hunk 5]
```diff
@@ -35,7 +32,7 @@ public function fetchData(): array
     {
         $response = $this->client->request(
             Request::METHOD_GET,
-            $this->httpImportAddress
+            ''
```
Comment (ovidiuenache) [inline src/HttpClient/HttpClientImportPogramme.php]: why empty string? The env var you define should not have the path. Just the domain and maybe a /api or something

**Review by ovidiuenache — CHANGES_REQUESTED** [review]
Please address the feedback above

File: config/packages/messenger.yaml (hunk 2 above)
Comment (marceldobrica) [reply config/packages/messenger.yaml:14]: Yes. I have tested. I no longer have any personal mailNotification and mailNotificationHandler, I'm using symfony's

Comment (marceldobrica) [reply config/packages/messenger.yaml:14]: I have tested it again now.

File: src/HttpClient/HttpClientImportPogramme.php (hunk 5 above)
Comment (marceldobrica) [reply src/HttpClient/HttpClientImportPogramme.php]: Ok. Changed value in .env.local

File: src/HttpClient/HttpClientSmsUser.php (hunk 4 above)
Comment (marceldobrica) [reply src/HttpClient/HttpClientSmsUser.php]: Ok. Changed value in env.local

File: src/HttpClient/HttpClientSmsUser.php (hunk 3 above)
Comment (marceldobrica) [reply src/HttpClient/HttpClientSmsUser.php]: Ok. Added in scoped clients configuration.

File: .env (hunk 1 above)
Comment (marceldobrica) [reply .env]: ok

**Pull request merged**